tests/bdd/scale/test_NAS_T1062.py

```diff
@@ -111,14 +114,22 @@ def click_the_start_automatically_ssh_checkbox_and_enable_the_ssh_service(driver
     value_exist = attribute_value_exist(driver, '//mat-slide-toggle[@ix-auto="slider__state__SSH"]', 'class', 'mat-checked')
     if not value_exist:
         driver.find_element_by_xpath('//div[@ix-auto="overlay__stateSSH"]').click()
-    time.sleep(1)
+    time.sleep(4)
 
 
 @then('the service should be enabled with no errors')
 def the_service_should_be_enabled_with_no_errors(driver):
     """the service should be enabled with no errors."""
-    wait_for_value = wait_for_attribute_value(driver, 7, '//mat-slide-toggle[@ix-auto="slider__state__SSH"]', 'class', 'mat-checked')
```

Comment:
I think this would be a better method than time.sleep(4).

Look at what I did for 22.02-RC.1 https://github.com/truenas/webui/blob/70409fe8a93b6af94d06c6e26d2b0c0d2d65fef9/tests/bdd/scale/test_NAS_T1062.py#L114-L121